types/chart.js/index.d.ts

```diff
@@ -36,6 +37,11 @@ declare class Chart {
 
     static defaults: {
         global: Chart.ChartOptions & Chart.ChartFontOptions;
+        [key: string]: any;
```

Comment:
Could you explain in more detail why this change is needed?

Reply:
If you want to extend an actual chart type on `chart.js`, you also need to create a default value for this chart, because of a bug. See [this doc](http://www.chartjs.org/docs/latest/developers/charts.html#extending-existing-chart-types) : 

> // Sets the default config for 'derivedBubble' to be the same as the bubble defaults. 
> // We look for the defaults by doing Chart.defaults[chartType]
> // It looks like a bug exists when the defaults don't exist
> Chart.defaults.derivedBubble = Chart.defaults.bubble;

Reply:
OK, thanks!